### PR TITLE
add bed and bed-version model

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
This PR adds bed and bed-version model to cg and orderportal

**How to prepare for test**:
- [x] run database migration (cg/scripts/add-target_caputer_bed.sql) on cg-stage database
- [x] install on stage of the hasta machine: `bash servers/resources/hasta.scilifelab.se/update-cg-stage.sh add-bed-to-sample`
- [x] install on stage of the clinical-db machine: `bash servers/resources/clinical-db.scilifelab.se/update-cgweb-stage.sh add-bed-to-sample
- [x] install on stage of the clinical-db machine: `bash servers/resources/clinical-db.scilifelab.se/update-clinical-api-stage.sh add-bed-to-sample

**How to test**:
- [x] login to OP-stage
- [x] create and MIP/balsamic order
- [x] submit the order
- [x] open the clinical-api-stage
- [x] find the submitted sample in the sample view

**Expected test outcome**:
- [x] The sample should have a bed-version
- [x] Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by @barrystokman 
- [x] tests executed by @patrikgrenfeldt 
- [x] "Merge and deploy" approved by @patrikgrenfeldt 
Thanks for filling in who performed the code review and the test!

This is minor **version bump** because it was created to be transparent to the user and the rest of the processes and is thus backwards compatible

### CODE REVIEW

**Comment by patrikgrenfeldt** [comment]
Test result:
image
👍

File: cg/server/api.py [hunk 1]
```diff
@@ -327,6 +327,8 @@ def options():
 
     source_groups = {'metagenome': METAGENOME_SOURCES, 'analysis': ANALYSIS_SOURCES}
 
+    print('beds', [bed.name for bed in db.beds()])
```
Comment (patrikgrenfeldt) [inline cg/server/api.py]:
```suggestion
```

File: tests/meta/orders/test_meta_orders_status.py [hunk 2]
```diff
@@ -374,6 +374,66 @@ def test_store_families_bad_apptag(orders_api, base_store, mip_status_data):
...
+    for case in cases:
+        assert case.links
+        for link in case.links:
+            assert not link.sample.bed_version
+
+
+def test_store_case_valid_capture_kit(orders_api, base_store, balsamic_status_data):
+    status_data = balsamic_status_data
+
+    # GIVEN a basic store with no samples or nothing in it + scout order
```
Comment (barrystokman) [inline tests/meta/orders/test_meta_orders_status.py]: Can you double check that the GIVEN - WHEN - THEN descriptions match the test case? They seem to be the same as the test case before.

Comment (patrikgrenfeldt) [reply tests/meta/orders/test_meta_orders_status.py]: well spotted! fixed now

**Comment by patrikgrenfeldt** [comment]
deployed

**Pull request merged**